examples/nuts-did.md

```diff
@@ -0,0 +1,89 @@
+The example below describes a generic Nuts DID to be used in the Nuts network:
```

Comment:
This example is no longer up2date with the spec. Since the spec has all examples embedded, remove this one?